content/codespaces/managing-your-codespaces/reviewing-your-security-logs-for-github-codespaces.md

```diff
@@ -15,7 +15,7 @@ redirect_from:
 
 ## About security logs for {% data variables.product.prodname_github_codespaces %}
 
-When you perform an action related to {% data variables.product.prodname_github_codespaces %} in repositories owned by your personal account, you can review the actions in the security log. For information about accessing the log, see "[AUTOTITLE](/authentication/keeping-your-account-and-data-secure/reviewing-your-security-log#accessing-your-security-log)."
+You can utilize the security log to review all actions related to {% data variables.product.prodname_github_codespaces %}. When you perform an action related to {% data variables.product.prodname_github_codespaces %} in repositories owned by your personal account, you can review the actions in the security log. For information about accessing the log, see "[AUTOTITLE](/authentication/keeping-your-account-and-data-secure/reviewing-your-security-log#accessing-your-security-log)."
```

Comment:
You can only use your security log to review actions related to codespaces in repositories owned by your personal account, not all actions related to GitHub Codespaces. Since that's already clearly indicated by the existing sentence, this new one is not necessary.

```suggestion
When you perform an action related to {% data variables.product.prodname_github_codespaces %} in repositories owned by your personal account, you can review the actions in the security log. For information about accessing the log, see "[AUTOTITLE](/authentication/keeping-your-account-and-data-secure/reviewing-your-security-log#accessing-your-security-log)."
```